### ISSUE TITLE
Lunchbox gui can't be translated

### ISSUE THREAD

**Issue by labpaw** [17cupsofcoffee/Still-Hungry#20]
Title says it all.

**Comment by 17cupsofcoffee** [comment]
Yeah, this is because the label on the lunchbox GUI is in the texture itself rather than being rendered by the game, kind of a short sighted decision we made before translations were possible in the mod. This will get fixed, don't worry :)
